CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Introduce new `LogLevel` constants, deprecating old ones.
 - Fix bug with linking process being stuck when developing on windows machines. [#471](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/471).
+- Fix crash on startup if `setAppSecret` method of `AppCenterReactNativeShared` was called.
```

Comment:
That's not a bug from last release though. It was introduced during development of this version.

Reply:
Thanks @guperrot. Let's remove this from changelog.